paraglobNode.cpp

```diff
@@ -0,0 +1,22 @@
+#include "paraglobNode.h"
```

Comment:
These methods are small enough that I think they could just move inline into the header. In fact, maybe move all the `ParaglobNode` stuff into the main `paraglob` header? Then it's just a single header file that's needed to use the library.

I'd also move everything internal (ParaglobNode) into an internal namespace to make clear it's not part of the public API.